### PR TITLE
Add iOS Microsurvey docs

### PR DESCRIPTION
**Pull request by cyndichin**
## Description
Add documentation for microsurveys and configuring the nimbus recipe on iOS.

## Issue that this pull request resolves
No specific experimenter issue. Related iOS [ticket](https://mozilla-hub.atlassian.net/jira/software/c/projects/FXIOS/boards/700?assignee=625c9c3cd7f1b80069bcb183&selectedIssue=FXIOS-9065).

## Other information
I have ran the project locally with my changes and it looks good, but please feel free to confirm as well!
| Screenshot | 
| --- | 
| image |

### CODE REVIEW

**Comment by cyndichin** [comment]
@adudenamedruby @jaredlockhart Please review and let me know if any changes to make. 

@t-p-white @Amejia481 I organized the docs a bit differently, let me know if there are any issues with this suggestion!

File: docs/workflow/implementing/messaging/microsurveys/ios-mobile-microsurveys.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+Note that this page was created during the MVP phase of the microsurvey project.
+
+## UI
+The microsurvey feature is composed of two main UI components. One is the prompt and one is the actual survey.
+
+### Prompt
+
+The prompt is an invitation to start the survey and it is shown when the triggers are met. The triggers are determined through the mobile messaging infrastructure and should be specified by the Nimbus recipe. For testing, we have created a trigger in which the user will be eligible to receive the survey after visiting the homepage twice.
+
+The prompt is shown until the user completes the survey, closes the survey or reached the maximum number of display specified in the mobile messaging configurations.
```
Comment (jaredlockhart) [inline docs/workflow/implementing/messaging/microsurveys/ios-mobile-microsurveys.md:18]: `reached` should be `reaches`

File: docs/workflow/implementing/messaging/microsurveys/ios-mobile-microsurveys.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+
+<img src="/img/messaging/microsurveys/ios-prompt.png" alt="Prompt view" className="img-sm"/>
+<br/><br/>
+
+1. This title of the prompt which can be customized, but also has a fallback string.
```
Comment (jaredlockhart) [inline docs/workflow/implementing/messaging/microsurveys/ios-mobile-microsurveys.md:23]: What is the fallback string?

File: docs/workflow/implementing/messaging/microsurveys/ios-mobile-microsurveys.md [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+2. The close button dismisses the survey and it won't be shown again.
+3. The continue button starts the survey and shows the survey sheet. It can be customized, but also has a fallback string.
```
Comment (jaredlockhart) [inline docs/workflow/implementing/messaging/microsurveys/ios-mobile-microsurveys.md:25]: There's a fallback string for the entire survey?  Also what is it?

**Review by jaredlockhart — APPROVED** [review]
Wow @cyndichin this is fantastic!!!!  Thank you for writing this up so comprehensively.  Let's land this and then if we need to make adjustments to it in the future we can.  Great job!

**Comment by mattreaganmozilla** [comment]
This is a minor nit but I'm not entirely clear on why we're checking in the image files into our main repo? Although they're not terribly large some of them are ~400kb or more and add a decent amount of binary data/overhead to our repository.

Wondering if there might be a better option for this, e.g. hosting these images externally, or storing them in Git LFS, etc.

File: docs/workflow/implementing/messaging/microsurveys/ios-mobile-microsurveys.md (hunk 1 above)
Comment (cyndichin) [reply docs/workflow/implementing/messaging/microsurveys/ios-mobile-microsurveys.md]: thanks for the catch, updated!

File: docs/workflow/implementing/messaging/microsurveys/ios-mobile-microsurveys.md (hunk 2 above)
Comment (cyndichin) [reply docs/workflow/implementing/messaging/microsurveys/ios-mobile-microsurveys.md]: updated with permalink

File: docs/workflow/implementing/messaging/microsurveys/ios-mobile-microsurveys.md (hunk 3 above)
Comment (cyndichin) [reply docs/workflow/implementing/messaging/microsurveys/ios-mobile-microsurveys.md]: This is for the button label, I updated to clarify.

**Comment by jaredlockhart** [comment]
@mattreaganmozilla We've been hosting images here without issue for some time, if we find a reason to move them we can file an issue and addres that separately 👍

**Pull request merged**